src/EditorFeatures/VisualBasicTest/CodeActions/ExtractMethod/ExtractMethodTests.vb

```diff
@@ -645,14 +645,12 @@ End Namespace", TestOptions.Regular.WithLanguageVersion(LanguageVersion.VisualBa
         [|Await Task.Delay(duration).ConfigureAwait(False)|]
     End Function
 End Class",
-"Imports System.Threading.Tasks
```

Comment:
these went away because there are unqualified names like `Task` already in the file, and adding this import could conflict with them.  the tests are a bit strange in that way because the original code is in error.

Reply:
It feels like here the better fix is just to add Imports System.Threading.Tasks to the input of the test, so that way we're actually ensuring the extract method works in the non-broken case. I'm guessing that was just an oversight when the test was written and should have been added originally.